mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
Should we introduce an 'internal heartbeat Rpc' to check if the server side is ready instead of relying on this?